test/plugins/individual-collateral/compoundv3/CusdcV3Wrapper.test.ts

```diff
@@ -639,15 +641,17 @@ describeFork('Wrapped CUSDCv3', () => {
       await advanceBlocks(1)
 
       expect(await compToken.balanceOf(bob.address)).to.be.greaterThan(0)
-      expect(await compToken.balanceOf(bob.address)).to.equal(
-        await compToken.balanceOf(don.address)
-      )
+      const expectedBalanceBob = await compToken.balanceOf(bob.address)
+      expect(expectedBalanceBob).to.be.closeTo(
+        await compToken.balanceOf(don.address),
+        expectedBalanceBob.mul(5).div(1000)
+      ) // within 0.5%
     })
 
     // In this forked block, rewards accrual is not yet enabled in Comet
-    it('claims no rewards when rewards accrual is not enabled', async () => {
+    // Only applies to Mainnet forks (L1)
+    itL1('claims no rewards when rewards accrual is not enabled', async () => {
```

Comment:
why don't we want to test this on base?

Reply:
Requires a fork block by default with rewards disabled. Could not find in Base. If there is one we can try.